docs/odata/v2vsv4.markdown

```diff
@@ -0,0 +1,48 @@
+  * `Edm.Float` has been eliminated.
+
+* The oData V4 model does not support the methods `getData`, `getObject`, `getOriginalProperty`, `getProperty`. For data access, use the context API instead of methods on the model.
+
+* The oData V4 model does not support the methods `getChangeBatchGroups`, `getChangeGroups`, `getDeferredGroups`, `setChangeBatchGroups`, `setChangeGroups`, `setDeferredBatchGroups`, `setDeferredGroups`, `setUseBatch` (and corresponding model construction parameters). Batch groups are solely defined via binding parameters with the corresponding parameters on the model as default. Application groups are by default deferred; there is no need to set or get deferred groups. You just need the `submitBatch` method on the model to control execution of the batch. You can use the predefined batch group `$direct` to switch off batch either for the complete model or for a specific binding (only possible for the complete model in V2).
+
+* OData operations executed via binding: The oData V4 model does not support the method `callFunction`. Use an operation binding instead.
+
+* No CRUD methods on model: The oData V4 model does not support the methods `create`, `read`, `remove`, `update`. `read`, `update`, `create` and `remove` operations are available implicitly via the bindings, so that changes are bound to controls. It is not possible to trigger requests for specific OData URLs.
+
+* No metadata access via model: The oData V4 model does not support methods `getServiceAnnotations`, `getServiceMetadata`, `refreshMetadata` as well as methods corresponding to the events `metadataFailed`, `metadataLoaded`. Metadata is only accessed via `ODataMetaModel`. Metadata is only loaded when needed (e.g. for type detection or to compute URLs for write requests); the corresponding methods on the `v4.ODataMetaModel` use promises instead of events.
+
+* `sap.ui.model.odata.AnnotationHelper` is not supported for OData V4.
+
+---
+
+## NOTE
+
+Due to the limited feature scope of this version of the SAPUI5 OData V4 model, check that all required features are in place before developing applications. Double check the detailed [documentation](https://help.sap.com/viewer/468a97775123488ab3345a0c48cadd8f/1809.000/en-US/e1b625940c104b558e52f47afe5ddb4f.html) of the features, as certain parts of a feature may be missing although you might expect these parts as given. Some controls might not work due to small incompatibilities compared to `sap.ui.model.odata.(v2.)ODataModel`, or due to missing features in the model (like tree binding). This also applies to SmartControls (`sap.ui.comp`) and Fiori Elements which do not support the SAPUI5 OData V4 model in general as well as to controls like TreeTable and AnalyticalTable which are not supported in conjunction with the SAPUI5 OData V4 model. The interface for applications has been changed for easier and more efficient use of the model.
```

Comment:
this has changed:
- FE is full v4 nowxx
- i'm not sure on the `sap.ui.comp` controls -> i hope they'll be adapted to v4 as well, but i have no knowledge on that. please follow up w/ e.g. @margopolo on thi

Reply:
This still seems to be valid according to the latest SAPUI5 documentation: https://sapui5.hana.ondemand.com/#/topic/5de13cf4dd1f4a3480f7e2eaaee3f5b8.html

Reply:
probably the docs lag behind...let's put this on friday's agenda and ask @margopolo